server/pull_request_handler.go

```diff
@@ -50,7 +50,10 @@ func (s *Server) pullRequestEventHandler(w http.ResponseWriter, r *http.Request)
 	switch event.Action {
 	case "opened":
 		mlog.Info("PR opened", mlog.String("repo", pr.RepoName), mlog.Int("pr", pr.Number))
-		if err = s.handleCheckCLA(ctx, pr); err != nil {
+
+		var claCommentNeeded bool
```

Comment:
Any reason why we cannot do `claCommentNeeded, err :=` ? The variable should still be available now that we have split the if condition.

Reply:
If we do that, `err` will shadow. It's just a linter think. I personally like the `var` style more then using a different name for `err`.

Reply:
We have been using `err2` in server repo. But this is also OK.